partitura/performance.py

```diff
@@ -280,6 +284,96 @@ def adjust_offsets_w_sustain(
         note["sound_off"] = offset
 
 
+class PerformedNote(dict):
+    def __init__(self, *args, **kwargs):
```

Comment:
I think that we should keep the same interface as in other similar classes and avoid using just `*args` and `**kwargs`.